src/internet_identity/src/storage.rs

```diff
@@ -396,6 +425,79 @@ impl<M: Memory + Clone> Storage<M> {
         }
     }
 
+    #[allow(dead_code)]
+    pub fn from_memory_v6_to_v7(memory: M) -> Option<Self> {
+        let maybe_storage_v6 = Self::from_memory(memory.clone());
+        maybe_storage_v6.as_ref()?;
```

Comment:
What is the purpose of this line?

Reply:
It checks whether `maybe_storage_v6` is `Some` yet without unwrapping (and returns `None` immediately if the value is `None`)

Reply:
TIL! Thanks, I thought it only worked on `Result`.